e2e/libs/volume/volume.py

```diff
@@ -49,8 +49,8 @@ def wait_for_volume_state(self, volume_name, desired_state):
         return self.volume.wait_for_volume_state(volume_name, desired_state)
 
     def wait_for_volume_attached(self, volume_name):
-        self.volume.wait_for_volume_state(volume_name, "attached")
         self.volume.wait_for_volume_robustness_not(volume_name, "unknown")
+        self.volume.wait_for_volume_state(volume_name, "attached")
```

Comment:
I assume, making sequence change here makes more sense :D

Reply:
The original sequence order could fail test cases in some conditions. Like the volume is automatically attached, detached and re-attached in a flash.